zkevm-circuits/src/copy_circuit.rs

```diff
@@ -791,7 +791,11 @@ pub mod dev {
         const NUM_BLINDING_ROWS: usize = 7 - 1;
         let instance = vec![vec![randomness; num_rows - NUM_BLINDING_ROWS]];
         let prover = MockProver::<F>::run(k, &circuit, instance).unwrap();
-        prover.verify()
+        let res = prover.verify();
+        if let Err(ref e) = res {
+            dbg!(e);
```

Comment:
You can get the same effect by doing `assert_eq(res, Ok(()))` instead of `assert_eq(res.is_ok())`